superset-frontend/src/datasource/ChangeDatasourceModal.tsx

```diff
@@ -47,86 +68,93 @@ const TABLE_COLUMNS = [
   'creator',
 ].map(col => ({ accessor: col, Header: col }));
 
-const TABLE_FILTERABLE = ['rawName', 'type', 'schema', 'connection', 'creator'];
 const CHANGE_WARNING_MSG = t(
   'Changing the dataset may break the chart if the chart relies ' +
     'on columns or metadata that does not exist in the target dataset',
 );
 
+const useDebouncedEffect = (effect: any, delay: number) => {
```

Comment:
Nit: let's move this to some `utils.ts` module, so it can be used by other components.